src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="http://localhost:8080/"
+    method="post"
+    class="bar-form"
+    name="bar-form">
```

Comment:
```suggestion
    <form
      action="http://localhost:8080/"
      method="post"
      class="bar-form"
      name="bar-form"
    >
```
check your code style

Reply:
check all such cases